### ISSUE TITLE
fatal error: module 'connectivity' not found @import connectivity  in GeneratedPluginRegistrant.m

### ISSUE THREAD

**Issue by vanessabin** [flutter/flutter#73554]
Hello all, 

I have now looked through all possible repositories and have not found a solution to this problem. 

The problem started when I installed the latest version of Android Studio (4.1.1). After the Flutter and Dart plugins did not work, I changed the channel to `Channel Stable` 

I get now again and again problems with the imported modules. First with the package barcode_scan. Now with the connectivity import.  

**I tried:** 
- Delete pod files from project
- Delete podfile.lock from project
- run pod install
- deleted the repository and cloned again

I also cleaned the project. 

**Flutter Doctor Output:** 
[✓] Flutter (Channel stable, 1.22.5, on Mac OS X 10.15.7 19H2 darwin-x64, locale de-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 12.3)
[✓] Android Studio (version 4.1)
[✓] Connected device (2 available)

• No issues found!
I cannot run the app on simulator or real device without getting this error: 
`Module 'connectivity' not found`

**When running pod install I get:** 
Analyzing dependencies
Downloading dependencies
Generating Pods project
Integrating client project
Pod installation complete! There are 0 dependencies from the Podfile and 0 total pods installed.

[!] The Podfile does not contain any dependencies.

[!] Automatically assigning platform `iOS` with version `12.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

[!] CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target `Runner` to `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` or include the `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` in your build configuration (`Flutter/Release.xcconfig`).

Could this cause the problem? 
I don't know how to solve it.

**Comment by pedromassangocode** [comment]
Hi @vanessabin 
Can you please provide your `flutter doctor -v`, your `flutter run --verbose` and your `pubspec.yaml`. 
Thank you

**Comment by vanessabin** [comment]
Sorry for the late response. 
I solved the problem. 
Maybe others are still facing the same problem, so here is what I did: 

- deleted my repository from computer
- deleted the project from Xcode (because it still showed the runner)
- cloned the project again
- deleted the Podfile.lock 
- deleted the Pods Folder
- run the following in the Project Folder in terminal: `rm ios/Podfile && flutter build ios`

**Comment by fldvlpr** [comment]
/Users/jaym/Documents/CLProjects/Flutter/SampleProjects/flutter_vm/ios/Runner/GeneratedPluginRegistrant.m:10:9: fatal error: module 'connectivity' not found
    @import connectivity;
     ~~~~~~~^~~~~~~~~~~~
    1 error generated.
    
    Nothing above worked

**Comment by heinzan** [comment]
Someone have been solved?

**Comment by chorauoc** [comment]
Hi,

I have the same issue, have anyone resolved this issue

Thanks
Asanka

**Comment by bobwith2bees** [comment]
> /Users/jaym/Documents/CLProjects/Flutter/SampleProjects/flutter_vm/ios/Runner/GeneratedPluginRegistrant.m:10:9: fatal error: module 'connectivity' not found
> @import connectivity;
> ~~~~~~~^~~~~~~~~~~~
> 1 error generated.
> 
> ```
> Nothing above worked
> ```

Once I made my Podfile platform ios version consistent with the Flutter MinimimOSVersion, and Xcode Target Build version, I was able to get a release build working.  Funny symptom, the 'flutter run' was able to build fine.  I had issues with 'flutter build ios' command when it was time to release.  I am confident there is a more elegant way to set the minimum target release in fewer places, but folks seemed desperate. ;-)

ios/Podfile:
platform :ios, '12.1'

ios/Flutter/AppFrameworkInfo.plist:
  <key>MinimumOSVersion</key>
  <string>12.1</string>

Xcode > Runner > Targets Runner > General > Deployment Info:
* Set IOS 12.1
